pom.xml

```diff
@@ -226,6 +226,12 @@
                 <groupId>org.wildfly</groupId>
                 <artifactId>wildfly-arquillian-container-managed</artifactId>
                 <version>${wildfly.version}</version>
+                <exclusions>
```

Comment:
fair enough with me, just wondering how this is related to HV-915 and Eclipse setup?